aws-datastore/src/main/java/com/amplifyframework/datastore/syncengine/Merger.java

```diff
@@ -105,10 +99,17 @@
                 // and the version would get bumped up.
                 .filter(currentVersion -> currentVersion == -1 || incomingVersion > currentVersion)
                 // If we should merge, then do so now, starting with the model data.
-                .flatMapCompletable(shouldMerge ->
-                        (isDelete ? delete(model, changeTypeConsumer) : save(model, changeTypeConsumer))
-                                .andThen(save(metadata, NoOpConsumer.create()))
-                )
+                .flatMapCompletable(shouldMerge -> {
+                    Completable firstStep;
+                    if (mutationOutbox.hasPendingMutation(model.getId())) {
+                        LOG.info("Mutation outbox has pending mutation for " + model.getId()
+                            + ". Saving the metadata, but not model itself.");
+                        firstStep = Completable.complete();
```

Comment:
What happens to any changes made to the model?

Reply:
They get dropped.  The item in the outbox wins over any model coming from the API.   However, we do still need to update the metadata, so that when the item in the outbox _does_ get processed, it doesn't get rejected because the `_version` is too old.